cli/src/commands/state/mod.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2023 -  Restate Software, Inc., Restate GmbH.
+// All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+mod edit;
+mod get;
+mod util;
+
+use cling::prelude::*;
+
+#[derive(Run, Subcommand, Clone)]
+pub enum ServiceState {
+    Get(get::Get),
+    Edit(edit::Edit),
```

Comment:
Please add a descriptive help messages for every command by adding a comment per variant. Clap will extract the comment automatically (see other commands for an example)